_posts/2020-05-30-newpipe-0.19.4-release.md

```diff
@@ -67,6 +67,8 @@ NewPipe also has its own F-Droid repository now! Read below for more info.
 
 ## Where to get the latest version
 
-NewPipe notifies you about new versions, you can download them when you press the notification. An alternative is the [GitHub release page](https://github.com/TeamNewPipe/NewPipe/releases). If you use the [F-Droid app](https://f-droid.org/), it notifies you as well about an update for [NewPipe](https://f-droid.org/packages/org.schabi.newpipe/) - please keep in mind that it can take F-Droid a while to update their repository. A new alternative to this is adding `https://archive.newpipe.net/fdroid/repo?fingerprint=E2402C78F9B97C6C89E97DB914A2751FDA1D02FE2039CC0897A462BDB57E7501` as a new repository in your F-Droid client then updating. You will get updates quickly and within the F-Droid app. These builds are the same as the ones released on GitHub, so if you currently have the official F-Droid version, you will need to uninstall that to use a GitHub/NewPipe repository build (make sure to back up data).
+NewPipe notifies you about new versions, you can download them when you press the notification. An alternative is the [GitHub release page](https://github.com/TeamNewPipe/NewPipe/releases). If you use the [F-Droid app](https://f-droid.org/), it notifies you as well about an update for [NewPipe](https://f-droid.org/packages/org.schabi.newpipe/) - please keep in mind that it can take F-Droid a while to update their repository. A new alternative to this is copying [this repository link](https://archive.newpipe.net/fdroid/repo?fingerprint=E2402C78F9B97C6C89E97DB914A2751FDA1D02FE2039CC0897A462BDB57E7501) and adding it as a new repository in your F-Droid client then updating. You will get updates quickly and within the F-Droid app. These builds are the same as the ones released on GitHub, so if you currently have the official F-Droid version, you will need to uninstall that to use a GitHub/NewPipe repository build (make sure to back up data).
 
 Please let us know what your experience of the latest release is, especially bugs in need of fixing. As usual, you can reach out to us via IRC ([#newpipe on freenode](https://webchat.freenode.net/?channels=newpipe)), [open issues on GitHub](https://github.com/TeamNewPipe/NewPipe/issues/new) or ideally use our built-in crash reporter to send us machine-readable issue reports. You can even [send in fixes](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md#bug-fixing) yourself.
+
+We need testers for new versions of NewPipe! If you have a KitKat, Android TV, or "battery optimized device", please add yourself to the relevant section of the [testers wiki page](https://github.com/TeamNewPipe/NewPipe/wiki/Testers) and you will be mentioned when you are needed.
```

Comment:
```suggestion
We need testers for new versions of NewPipe! If you have a KitKat, Android TV, or "battery optimized" device, please add yourself to the relevant section of the [testers wiki page](https://github.com/TeamNewPipe/NewPipe/wiki/Testers) and you will be mentioned when you are needed.
```
because a kikat doesn't make sense